src/TypesAndInterfaces/config-sync/input-config-sync/profile-field-config.ts

```diff
@@ -65,6 +65,17 @@ export const getDateYearsAgo = (years: number = 13):Date => {
     return date;
 }
 
+export const walkLevelMultiplier:number = 2; //Database range 1-10
+
+export const walkLevelOptions: Map<number, [string, string]> = new Map<number, [string, string]>([
+    [1, ['😟', 'Distant and Disconnected']],
```

Comment:
I'm curious, why'd you reverse the order?

Reply:
I did that when I was working with the numbered order when I was still using the slider so that it would list the items in ascending order. Now that we aren't I don't need to. I can put it back